services/github/github-release-date.service.js

```diff
@@ -49,61 +51,43 @@ module.exports = class GithubReleaseDate extends LegacyService {
           user: 'Cockatrice',
           repo: 'Cockatrice',
         },
-        staticPreview: {
-          label: 'release date',
-          message: 'december 2018',
-          color: 'green',
-        },
+        staticPreview: this.render({ date: '2017-04-13T07:50:27.000Z' }),
         documentation,
       },
     ]
   }
 
-  static registerLegacyRouteHandler({ camp, cache, githubApiProvider }) {
-    camp.route(
-      /^\/github\/(release-date|release-date-pre)\/([^/]+)\/([^/]+)\.(svg|png|gif|jpg|json)$/,
```

Comment:
For the moment I haven't changed the URLs, but `/release-date-pre` should probably move to `/release-date?include_prereleases=true` as we progress the work discussed in #3144